README.md

```diff
@@ -1,36 +1,43 @@
-# UnitManager  
-UnitManager is a GUI tool that helps you manage the unit cells of Personium.  
-No installation or compilation, you only need to upload the UnitManager source files to your web server.  
+# Cell Manager  
+Cell Manager is a GUI tool that helps you manage Personium cell(s).  
 
-## Installation    
+- Cells within a Personium Unit  
+- Cell belong to a user/organization  
 
-### Prerequisites  
-Before you install and use the UnitManager, make sure you have the following information ready.  
+## Prerequisites  
+In order to use the Unit/Cell Manager, make sure you have the following ready.  
```

Comment:
Is the description of [Unit/Cell Manager] intentional?
Is there a problem with [Cell Manager]?

Reply:
It should be Cell Manager.

Reply:
I confirmed that it has been corrected.
Thank you.